ast/location/location.go

```diff
@@ -10,10 +10,11 @@ import (
 
 // Location records a position in source code
 type Location struct {
-	Text []byte `json:"-"`    // The original text fragment from the source.
-	File string `json:"file"` // The name of the source file (which may be empty).
-	Row  int    `json:"row"`  // The line in the source.
-	Col  int    `json:"col"`  // The column in the row.
+	Text   []byte `json:"-"`    // The original text fragment from the source.
+	File   string `json:"file"` // The name of the source file (which may be empty).
+	Row    int    `json:"row"`  // The line in the source.
+	Col    int    `json:"col"`  // The column in the row.
+	Offset int    `json:"-"`    // The byte offset for the location in the source.
```

Comment:
Was mostly to avoid adding it to the like REST API and CLI responses, if I could have (without more intrusive change) I would have not added it to the public Golang API either.

If you think it would be useful to have in the JSON format I'm game to add it. Just didn't want to commit us to exposing it without any particular external use-case.

Reply:
No, that's fine. I just wanted to understand the reason for not exposing it for now. Thanks!